### PR TITLE
s/hightlight/highlight

### PR DESCRIPTION
**Pull request by PhilETaylor**
s/hightlight/highlight

### CODE REVIEW

File: build/media_source/system/js/fields/joomla-field-fancy-select.w-c.es6.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ window.customElements.define('joomla-field-fancy-select', class extends HTMLElem
     // Handle typing of custom Term
     if (this.allowCustom) {
       // START Work around for issue https://github.com/joomla/joomla-cms/issues/29459
-      // The choices.js always auto-hightlight first element
+      // The choices.js always auto-highlight first element
```
Comment (brianteeman) [inline build/media_source/system/js/fields/joomla-field-fancy-select.w-c.es6.js]:
```suggestion
      // The choices.js always auto-highlights the first element
```

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 7506a84020e3a7d253d6313a2433087c4ee1b460<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33158">issues.joomla.org/tracker/joomla-cms/33158</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 7506a84020e3a7d253d6313a2433087c4ee1b460<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33158">issues.joomla.org/tracker/joomla-cms/33158</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33158">issues.joomla.org/tracker/joomla-cms/33158</a>.</sub>

**Pull request merged**